### PR TITLE
Add 'RawMessage' plugin channel

### PR DESCRIPTION
**Pull request by maxzz88**
Channel 'Message' is really old.

### CODE REVIEW

File: proxy/src/main/java/net/md_5/bungee/connection/BungeePluginChannelHandler.java [hunk 1]
```diff
@@ -126,6 +126,12 @@ private void handleMessage(DataInput in, ByteArrayDataOutput out) throws Excepti
         if ( target == null ) { return; }
         target.sendMessage( new TextComponent( in.readUTF() ) );
     }
+    
+    private void handleRawMessage(DataInput in, ByteArrayDataOutput out) throws Exception {
```
Comment (krillr) [inline proxy/src/main/java/net/md_5/bungee/connection/BungeePluginChannelHandler.java]: Any chance I could get you to move this down under handlePlayerList so the functions remain in properly alphabetical order? Makes it easier to find what you're looking for in the file. (I'm a bit OCD about this, LOL)

**Comment by krillr** [comment]
What are the advantages of using this over the original Message subchannel, and is there a good reason this new implementation couldn't just replace the old implementation? Would it break backwards compatibility with other plugins? (Not familiar with the TextComponent, ComponentSerializer, etc yet).

File: proxy/src/main/java/net/md_5/bungee/connection/BungeePluginChannelHandler.java (hunk 1 above)
Comment (maxzz88) [inline proxy/src/main/java/net/md_5/bungee/connection/BungeePluginChannelHandler.java]: Oh, ok.

Comment (krillr) [inline proxy/src/main/java/net/md_5/bungee/connection/BungeePluginChannelHandler.java]: Thanks :)

**Comment by maxzz88** [comment]
If we replace old method, we will break some plugins.
Here some examples, old method:

``` java
out.writeUTF("Message");
out.writeUTF("Player");
out.writeUTF("§aHello World!"); // Text
```

and new method:

``` java
out.writeUTF("RawMessage");
out.writeUTF("Player");
out.writeUTF("{\"text\":\"Hello World!\",\"color\":\"red\"}"); // Json
```

**Comment by krillr** [comment]
Ah! Ok :)

**Pull request merged**